src/main/java/ua/softserveinc/tc/constants/UserConstants.java

```diff
@@ -2,6 +2,8 @@
 
 
 public final class UserConstants {
+  public static final String USER_CALENDAR = "user-calendar";
+
```

Comment:
Take a look on the class of mine . 

There is two type of constans. One is our URL adress and another is our view name.